packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.module.css

```diff
@@ -35,6 +35,7 @@
 }
 
 :global(.webchat-fluent) .pre-chat-message-activity__card-action-image {
+  --monochrome-image-masker-color: var(--webchat-colorNeutralForeground4);
```

Comment:
@OEvgeny should I use a new `:global` here? And how?

Reply:
You may want to add `styleOptions`-like global variable something like `--webchat__mask-color--card-action-image`